### PR TITLE
Mina's branch

### PR DESCRIPTION
**Pull request by FPStruck**
Multi-Session Login System implementation

### CODE REVIEW

**Review by MassoniMacaroni — COMMENTED** [review]
Changes to ./classpath make the application unable to run and dashboard.fxml has had its css location updated incorrectly.

Will approve and fix with a new pull request shortly as I'd like Mina's changes to be in the work im about to do.

File: UHDD/src/application/UI.java [hunk 1]
```diff
@@ -367,7 +367,7 @@ protected PasswordAuthentication getPasswordAuthentication() {
 		} 
 		else {
 			if(loginSuccessful() == true) {
-				Parent root = FXMLLoader.load(getClass().getResource("\\fxmlScenes\\Dashboard.fxml")); // change to dashboard
+				Parent root = FXMLLoader.load(getClass().getResource("\\fxmlScenes/Dashboard.fxml")); // change to dashboard
```
Comment (TitanSri) [inline UHDD/src/application/UI.java:370]: Not sure if this link will work "/" should be "\\"

File: UHDD/src/application/DashboardController.java
Comment (TitanSri) [inline UHDD/src/application/DashboardController.java:1]: Duplicate from view controllers folder

File: UHDD/src/application/PatientInfoViewController.java
Comment (TitanSri) [inline UHDD/src/application/PatientInfoViewController.java:1]: Duplicate from view controllers folder

**Review by TitanSri — COMMENTED** [review]
A few duplication files:
Dashboard controller
Patient Info View controller
pom.xml

Patient Info View 3 and 4 are not required.

File: pom.xml
Comment (TitanSri) [inline pom.xml:1]: Duplicate from UHDD folder

**Review by TitanSri — COMMENTED** [review]
Duplicate pom.xml file

**Pull request merged**